### PR TITLE
feat(api): switch to swagger-ui

### PR DESCRIPTION
**Pull request by iawia002**
**What this PR does / why we need it**:

78764679-7a8a0e00-79b9-11ea-87e5-56addf10c425

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @supereagle @hezhizhen 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/nirvana/api/api.go [hunk 1]
```diff
@@ -204,74 +204,41 @@ func (o *apiOptions) pathForVersion(version string) string {
 
 func (o *apiOptions) indexData(versions []string) ([]byte, error) {
 	index := `
-{{ $total := len .}}
-{{ $multipleVersions := gt $total 1 }}
+<!-- HTML for static distribution bundle build -->
 <!DOCTYPE html>
-<html>
+<html lang="en">
```
Comment (hezhizhen) [inline cmd/nirvana/api/api.go:209]: 万一有中文咋办？

File: cmd/nirvana/api/api.go [hunk 2]
```diff
@@ -280,40 +247,36 @@ func (o *apiOptions) indexData(versions []string) ([]byte, error) {
...
 	tmpl, err := template.New("index.html").Parse(index)
 	if err != nil {
 		return nil, err
 	}
-	data := []struct {
+	data := make([]struct {
 		Name string
 		Path string
-	}{}
+	}, 0, len(versions))
```
Comment (hezhizhen) [inline cmd/nirvana/api/api.go:279]:
```
data := make([]xxx, len(versions))
for i, v := range versions {
    data[i].Name = v
    data[i].Path = o.pathForVersion(v)
}
```

File: cmd/nirvana/api/api.go (hunk 1 above)
Comment (iawia002) [reply cmd/nirvana/api/api.go:209]: This actually doesn't mean you can't write Chinese in the HTML file, this attribute just tells the browser that the language of this page is English, and then the browser can do something based on this information, eg: auto-translate.

**Comment by hezhizhen** [comment]
LGTM @supereagle

**Pull request merged**